api/dotnet/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+[package]
+name = "slint-dotnet"
+description = "Internal Slint Runtime Library for .NET API."
+authors.workspace = true
+documentation.workspace = true
+edition.workspace = true
+homepage.workspace = true
+keywords.workspace = true
+license.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+version.workspace = true
+categories = ["gui", "development-tools"]
+build = "build.rs"
+
+[lib]
+path = "src/lib.rs"
+crate-type = ["lib", "cdylib", "staticlib"]
+
+[dependencies]
+async-std = { version = "1.10.0" }
```

Comment:
You can also use the more lightweight spin-on crate for this (block_on). We use that in other places as well.